src/main-menu.ts

```diff
@@ -17,8 +18,22 @@ export class MainMenu extends LitElement {
 
   @state()
   private settingsOpen = false;
+  @state()
+  private loadState: "loading" | "loaded" | "done" = "loading";
```

Comment:
What's the difference between loaded and done? If it's for the click to allow audio playback thing let's leave a comment